elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -0,0 +1,182 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: navigation;
+
+:host {
+  --pfe-navigation__overlay--BackgroundColor: var(--pfe-theme--overlay--BackgroundColor, rgba(37, 37, 37, 0.5));
+
+  // Default spacing values
+  --pfe-navigation--Padding--vertical: #{pfe-var(container-padding)};
+  --pfe-navigation--Padding--horizontal: calc(#{pfe-var(container-padding)} * 2);
+
+  --pfe-navigation--Padding: 0 #{pfe-local(Padding--horizontal)};
+
+  // Darkest is the default background color
+  --pfe-navigation--BackgroundColor: #{pfe-color(surface--darkest)};
+  --pfe-navigation--Color: #{pfe-color(surface--darkest--text)};
+  --pfe-navigation--Border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} transparent;
+  --pfe-navigation--MinHeight: 72px;
+
+  // Reduce padding for print
+  @media print {
+    --pfe-navigation--Padding: calc(#{pfe-local(Padding--vertical)} / 2) #{pfe-local(Padding--horizontal)};
+  }
+
+  display: block;
+  z-index: 2;
+  width: 100%;
+}
+
+:host(.sticky) {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  z-index: 99;
```

Comment:
Would it be easy to override this z-index value in a situation where something is higher on the page like an alert?

Reply:
`z-index: var(--pfe-z-index--nav, 99);`

Reply:
Maybe we add z-indexing as a map in our pfe-sass like we do for webrh?

Reply:
Added this!  Thanks!